process/consensus.md

```diff
@@ -40,6 +42,62 @@ In some cases where consensus seems to obviously have been reached, the chair
 may poll a lighter-weight request for objections to unanimous consensus. A
 single objection is sufficient to force the above consensus process.
 
+## Online consensus
+
+It it critical that work progress between in-person meetings: agreed-upon
+designs need to move forward, and new ideas need to reach some level of maturity
+before being discussed in-person. To that end, this group can reach consensus
+online, either on GitHub repositories under the WebAssembly organization or in
+official video calls. In the later case, decisions are recorded in meeting notes
```

Comment:
do we really want to declare consensus on these calls? if so, how is consensus determined?